packages/mermaid/src/docs/syntax/sequenceDiagram.md

```diff
@@ -130,6 +130,15 @@ sequenceDiagram
     Note over Alice,John: A typical interaction
 ```
 
+It is also possible to add a line break (applies to text input in general):
+
+```mermaid-example
+sequenceDiagram
+    Alice->John: Hello John, how are you?
+    Note over Alice,John: A typical interaction<br>But now in two lines
```

Comment:
```suggestion
    Note over Alice,John: A typical interaction<br/>But now in two lines
```

Is it worth using a `<br/>` instead of a `<br>`?

Because Mermaid diagrams render as SVG, which is XML, some SVG tools throw errors on valid HTML like `<br>`, which aren't valid XML.